### PR TITLE
Add build_docs step to travis

### PR DESCRIPTION
**Pull request by 97amarnathk**
Command `python setup.py build_docs` works now

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,6 +53,8 @@ install:
 
 script:
    - python setup.py $SETUP_CMD
+   - if [[ $SETUP_CMD == 'build_docs' ]]; then chmod +x deploy_docs.sh; fi
```
Comment (wkerzendorf) [inline .travis.yml]: just use `sh` or `source` no need to modify the flags for the file

File: deploy_docs.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# create local env
+conda create -n python36 python=3.6 --yes
+
+source activate python36
+
+conda install -c conda-forge doctr --yes
+
+doctr deploy --no-require-master . --built-docs build/sphinx/html
```
Comment (wkerzendorf) [inline deploy_docs.sh]: why `no-require-master` - what does that do?

Comment (97amarnathk) [reply deploy_docs.sh]: It allows docs to be generated even from non master branches

Comment (wkerzendorf) [reply deploy_docs.sh]: That would re-generate the main documentation even for PRs - which is not what we want. We want to review the documentation before it goes live.

**Pull request merged**